meson.build

```
@@ -0,0 +1,183 @@
+project( 'apexpy', 'c',
+  # Note that the git commit hash cannot be added dynamically here
+  version: '1.1.0',
```

```suggestion
  version: '2.0.0',
```
When I build locally (`pip install .`), this is the version pip says I am installing.